core/src/main/java/org/apache/iceberg/BaseTransaction.java

```diff
@@ -107,6 +107,10 @@ public String tableName() {
   }
 
   public TableMetadata startMetadata() {
+    return base;
+  }
+
+  public TableMetadata currentMetadata() {
```

Comment:
Can we focus on the REST catalog pieces and not transaction pieces right now? I think there's plenty of work in updating the REST spec and adding the implementation.

Reply:
I'm fine with this being here. I was thinking about how to write the tests and this is an easy way to stack up multiple changes.